src/states/elder/mod.rs

```diff
@@ -664,6 +735,7 @@ impl Elder {
             first_state_serialized: self.gen_pfx_info.first_state_serialized.clone(),
             first_ages: self.gen_pfx_info.first_ages.clone(),
             latest_info: self.chain.our_info().clone(),
+            parsec_version: *self.gen_pfx_info.first_info.version(),
```

Comment:
Yeah, I agree it shall use `self.gen_pfx_info.parsec_info`
There is pruning happens in tests. 
I guess the soak tests passes because this is `starting version` of the parsec instances, and every nodes can having independ version-indexing.